proxy.ini

```diff
@@ -7,6 +7,21 @@ override-soa_edit_api = INCEPTION-INCREMENT
 override-nameservers = ns1.example.com. ns2.example.com. ns3.example.com. ns4.example.com.
 override-kind = Master
 
+[ldap]
+enabled = True
```

Comment:
I'd say to keep LDAP disabled by default, to remain backwards compatible.

Reply:
I changed it to default to false and also merged your latest commits into my project.